frontend/src/types/UserTypes.ts

```diff
@@ -1,5 +1,23 @@
+import { Role } from "./AuthTypes";
+
 export type AdminUser = {
   firstName: string;
   lastName: string;
   email: string;
 };
+
+export type UserRequest = {
```

Comment:
nit: `UserRequest` -> `NewUserRequest` or `AddUserRequest` or `RegisterUserRequest` (or something to that effect, I don't love any of those names LOL)